### PR TITLE
Adding files back

### PR DESCRIPTION
**Pull request by cybearg12**
Look at the code, the code is amazing!

### CODE REVIEW

File: indexBasecodeANSWER1c.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<!doctype html>
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:1]:
```suggestion
<!Doctype html>
```
I think we will be needing an Uppercase for `Doctype` definitions

File: design2.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
+body {
```
Comment (cybearg12) [inline design2.css:1]: The file names are not very intuitive, they would benefit from renaming something related to the project.

File: design2.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
+body {
+	background-color: #008800;
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	text-align: center;
+}
+* {
```
Comment (Karthik-Y) [inline design2.css:6]: Good usage of global pattern but it is generally good practice to avoid it. So we can prevent some of the pitfalls when scaling the styles.

File: design2.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+}
+html {height:100%; margin-bottom:1px;} /* Forces vertical scroll bar to be visible */
+#wrapper {
+	width: 900px;
+	margin-right: auto;
+	margin-left: auto;
+	background-image: url(images/Column1.jpg);
+	background-repeat: repeat-y;
+	text-align: left;
+	background-color: #FFFFFF;
```
Comment (Karthik-Y) [inline design2.css:18]: This is a minor information and no change needed, `#FFFFFF` is all uppercase letters the default Dom handles small case letters but modern browsers can easily translate this so not a problem.

File: design2.css (hunk 2 above)
Comment (cybearg12) [inline design2.css:1]: The files are very large, more than 100 lines each and it makes reading and maintaining very hard. I think the code base would benefit from splitting out these css and html files in smaller chunks.

File: design2.css [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+* {
+	margin: 0px;
+	padding: 0px;
+}
+html {height:100%; margin-bottom:1px;} /* Forces vertical scroll bar to be visible */
+#wrapper {
+	width: 900px;
+	margin-right: auto;
+	margin-left: auto;
+	background-image: url(images/Column1.jpg);
```
Comment (Karthik-Y) [inline design2.css:15]: the `url(images/Column1.jpg); ` is missing a quote around some browsers aren't capable of parsing this again some recent one does.  we can have this rewritten as `url('images/Column1.jpg'); `

File: design2.css [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+	background-repeat: repeat-y;
+	text-align: left;
+	background-color: #FFFFFF;
+}
+#mainphoto {
+	background-image: url(images/Main_photo1.jpg);
```
Comment (Karthik-Y) [inline design2.css:21]: we have a missing quote here..

File: design2.css [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+#col1content {
+	padding: 0px 15px 15px;
+	
+}
+#footer {
+	clear: both;
+	height: 25px;
+	width: 900px;
+	background-image: url(images/Footer.jpg);
```
Comment (Karthik-Y) [inline design2.css:77]: and here..

File: design2.css [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+#col1 {
+	float: left;
+	width: 650px;
+}
+#col2 {
+	float: left;
+	width: 250px;
+}
+#col1content p {
```
Comment (cybearg12) [inline design2.css:54]: I am noticing a lot of duplication in defining the styles. There are other ways of defining css classes which share some styles without relying on duplication and I feel the code can benefit from these. Would it be ok to organise these css styles into classes for each element in the page ?

File: design2.css [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+}
+h2 {
+
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	font-size: 20px;
+	color: #B4AA9E;
+	padding-top: 10px;
+	padding-right: 0px;
+	padding-bottom: 0px;
+	padding-left: 15px;
```
Comment (Karthik-Y) [inline design2.css:101]: What do you think using 
```
h1, h2 {
  ... common attributes
}

h1 {
  specific merits
}

h2 {
 specific merits
}
```
is more maintainable?

File: indexBasecodeANSWER1c.html (hunk 1 above)
Comment (cybearg12) [reply indexBasecodeANSWER1c.html:1]: The name of the index file in this scenario does not help to understand what that file is for. I feel this would work much better if it was named index.html and the website can render this by default when the user navigates to this folder in the server.

File: indexBasecodeANSWER1c.html [hunk 10]
```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+		<title>Flights to Mars</title>
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:5]: This is an information, we could have used a linter here that will align the tags in the right position after self closing tags like the <meta ../> above.

File: indexBasecodeANSWER1c.html [hunk 11]
```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
```
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:2]: The size of this page is too large and makes it very difficult to follow. Would it be better to split this into several html pages which can get imported into the main index html page?

File: indexBasecodeANSWER1c.html [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+    <link href="design2.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+      <br>
+      <div id="wrapper">
+            <div id="mainphoto">&nbsp;</div>  <!-- end mainphoto -->
+            <div id="nav">
+                  <table id='table1' width='900'>
+                  <tr>
+                  <td align='center' width='110' >
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:15]: ` >` linters will solve some of the spacing problems in the syntax. here is one for Vscode editors and I believe there are other similar to them in google for different IDEs  [HTML lint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint)

File: indexBasecodeANSWER1c.html [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+                  <img src='images\mercury.jpg' alt='This is Mercury' height='100' id='mercury'>
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\venus.jpg' alt='This is Venus' height='100' id='venus' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\moon.jpg' alt='This is Moon' height='100' width='100' id='moon' >
+                  </td>
+                  </tr>
+                  <tr>
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:34]: The indentation could helps us read the code better, again linters ☝️ will help us from manual editing.

**Review by MazH-26 — COMMENTED** [review]
The files for 'images/column' and 'images/column1' may have been easier to implement as a hexcode value in the CSS file.
As using images is extra calls to the sever and more overhead, which would mean the website would load slower.

File: indexBasecodeANSWER1c.html [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+                  Phone, Email and Credit Card details) and we will rush you
+                  the tickets. This flight is SO NOT likely to happen !!!!!! (for a while)</p>
+                  </div>  <!-- end col2content -->
+            </div>  <!-- end col2 -->
+            <div id="footer">
+            <p><b>&copy; Bear Computer Services Ltd. | Original style design by BlueSky Web Studios</b></p>
+            </div>  <!-- end footer -->
+      </div>  <!-- end wrapper -->
+</body>
+<script>
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:87]: Good work here I would have been more intrigued to apply a modular approach and keep javascripts in separate folder and load them when needed.

File: design2.css (hunk 2 above)
Comment (ashcleave) [inline design2.css:1]: I would suggest renaming this file to `index.css` seems reasonable because some of the sellectors (such as `body`, `html` and `*`) are typically used as part of a base .css file, often refered to as `index.css`.

File: indexBasecodeANSWER1c.html (hunk 12 above)
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:15]: The way images are loaded in this page in the page is very repetitive. We could look at improving this and simplifying the code by adding all the images to an array and using a map function to generate the table. This would make the code smaller and easier to understand.

File: indexBasecodeANSWER1c.html [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+		 function Init()
```
Comment (MazH-26) [inline indexBasecodeANSWER1c.html]: It's really good to see all of the code together, but it may be easier to read and maintain if the JavaScript and HTML were stored in separate files.

File: indexBasecodeANSWER1c.html [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+		 {
+		  	var ins = document.getElementsByTagName("input");
+		    var imgs = document.getElementsByTagName("img");
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:91]: And one of the good approaches of having modular like mentioned above it is easier on the linters without having them parse different languages in one file. Seems the indentation here can be avoided if we get JS linters. Here is [prettier](https://github.com/prettier/prettier-vscode)

File: indexBasecodeANSWER1c.html [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\venus.jpg' alt='This is Venus' height='100' id='venus' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\moon.jpg' alt='This is Moon' height='100' width='100' id='moon' >
+                  </td>
+                  </tr>
+                  <tr>
+                  <td id='marsT' align='center' width='110' >&nbsp;</td>
```
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:35]: The 'align' and 'width' attributes are repeated for every element, could benefit from defining this in a reusable css style.

File: indexBasecodeANSWER1c.html [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+			if (e.type == "mouseover" || e.type == "onmouseover"){
+		 		 document.getElementById(this.id + "T").innerHTML = document.getElementById(this.id).alt;
+			} else {
+				 document.getElementById(this.id + "T").innerHTML = "&nbsp;";
+			}
+		}
+		
+	 Init()
+
+</script>
```
Comment (MazH-26) [inline indexBasecodeANSWER1c.html:156]: It's really good to see all of the code together, but it may be easier to read and maintain if the JavaScript and HTML were stored in separate files.

File: indexBasecodeANSWER1c.html [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+                  <td id='venusT' align='center' width='110' >&nbsp;</td>
+                  <td id='moonT' align='center' width='110' >&nbsp;</td>
+                  </tr>
+                  </table>
+            </div> <!-- end nav -->
+            <div id="col1">
+                  <div id="col1content">
+                        <h3 id='orderform'>Flights to Mars - Order Form</h3>
+                        <p id='details'>
+                        <form method=post action="jcform.asp" name="edgar" >
```
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:48]: Could we move this form to  a separate file ?

File: design2.css (hunk 6 above)
Comment (ashcleave) [inline design2.css:21]: A clearer name for `images/Main_photo1.jpg` would probably help with readability of the code. Perhaps assign the string to a more descriptive variable name and pass that as the url argument function instead

File: indexBasecodeANSWER1c.html [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+            </div>  <!-- end footer -->
+      </div>  <!-- end wrapper -->
+</body>
+<script>
+		 function Init()
+		 {
+		  	var ins = document.getElementsByTagName("input");
+		    var imgs = document.getElementsByTagName("img");
+		 
+	 		if (window.addEventListener)
```
Comment (MazH-26) [inline indexBasecodeANSWER1c.html:93]: Good to see use of If statements and For loops, but it would be good to keep it consistent with have the Curly braces at the end of the statement line, like you have done on the For loops.

File: indexBasecodeANSWER1c.html [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+	 			document.getElementById("blurb").addEventListener("mouseout",doBackground,false);
+				
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].addEventListener("mouseover",doText,false);
+	 					imgs[i].addEventListener("mouseout",doText,false);
+				}
+	 		}
+	 		else if (window.attachEvent)
+	 		{
+	 			for(i=0 ; i<ins.length ; i++){
```
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:110]: I can't figure out what the name of the variables mean. It would make the code clearer if we used full words for our variable names.

File: indexBasecodeANSWER1c.html [hunk 22]
```diff
@@ -0,0 +1,157 @@
...
+	 			document.getElementById("blurb").attachEvent("onmouseover",doBackground);
+	 			document.getElementById("blurb").attachEvent("onmouseout",doBackground);
+				for(i=0 ; i<imgs.length ; i++){
+	 					imgs[i].attachEvent("mouseover",doText);
+	 					imgs[i].attachEvent("mouseout",doText);
+				}
+	 		}
+	 		else
+	 		{
+ 			 	alert("I have nothing");
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:123]: 😄 Seems we are alerting for `nothing` here. The use case will be for when pressing the `buy` button when we have nothing and this will be appropriate on the validation code for the buy form I guess.

File: indexBasecodeANSWER1c.html [hunk 23]
```diff
@@ -0,0 +1,157 @@
...
+		function doHighlights(e)
+		{
+			if (!e) { e = window.event; }
+			if (e.type == "focus" || e.type == "onfocus"){
+		 		 document.getElementById(this.id).style.backgroundColor = "red";
+			} else {
+				 document.getElementById(this.id).style.backgroundColor = "white";
+			}
+		}
+		function doBackground(e)
```
Comment (cybearg12) [inline indexBasecodeANSWER1c.html:135]: The name of the methods does not convey a lot of information about what they are meant to do. The code would benefit from giving more meaningful name to the methods.

File: indexBasecodeANSWER1c.html [hunk 24]
```diff
@@ -0,0 +1,157 @@
...
+	 		else
+	 		{
+ 			 	alert("I have nothing");
+	 		}
+	 	}
+		function doHighlights(e)
+		{
+			if (!e) { e = window.event; }
+			if (e.type == "focus" || e.type == "onfocus"){
+		 		 document.getElementById(this.id).style.backgroundColor = "red";
```
Comment (Karthik-Y) [inline indexBasecodeANSWER1c.html:130]: `this.id` can be better represented here as `e.target.id` as we have a functional styles here and passing in the element rather than catching the trigger of this.

File: indexBasecodeANSWER1c.html [hunk 25]
```diff
@@ -0,0 +1,157 @@
...
+                              <input type=Radio name="CC" value="Visa" checked>Visa
+                              <input type=Radio name="CC" value="Amex">Amex
+                              <input type=Radio name="CC" value="Mastercard">Mastercard
+                              </p>
+                              <p>Credit card number
+                              <input type=Password id="cardnumber" >
+                              </p>
+                              <p><nobr><input type='submit' value="Buy">
+                              <input type='reset' value="Clear"></nobr></p>
+                        </form></p>
```
Comment (MazH-26) [inline indexBasecodeANSWER1c.html:67]: Good layout of the form but it could be a good idea to have some validation implemented, so we aren't receiving incomplete records or data in an unexpected format.

File: design2.css [hunk 26]
```diff
@@ -0,0 +1,136 @@
...
+#col1content td {
+	margin-bottom: 10px;
+	color: #404040;
+	font-size: 12.5px;
+	line-height: 140%;
+}
+
+#col1content {
+	padding: 0px 15px 15px;
+
```
Comment (ashcleave) [inline design2.css:71]: Random line here to be removed 🙂

**Review by cybearg12 — COMMENTED** [review]
Good first iteration to get the page working! Great use of css and page events! 

I feel the size of the css and html files are too large which makes the code very hard to follow and also write tests for. 
There are vast improvements for maintainability to be gained by: 
- splitting the files into smaller chunks and following the Single Responsability Principle for each file
- moving the js to a separate set of files outside the page markup.
- generating the markup by using a map would reduce the size of of the code, removes repetition and would make it more maintainable 
- rename methods to use get/set/delete to make them more descripttive
- use css classes to compact the size of the css and reduce repetition between the style.

A bit of more work to be done but I feel it will get there with a bit of refactoring!

**Review by ashcleave — COMMENTED** [review]
Some comments looking specifically at readability

File: indexBasecodeANSWER1c.html [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+                              <p id='pt3'> Phone Number
+                              <input type=text id="phone" ></p>
+                              <p id='pt4'> Email
+                              <input type=text id="email" ></p>
+                              <p>Credit card type
+                              <input type=Radio name="CC" value="Visa" checked>Visa
+                              <input type=Radio name="CC" value="Amex">Amex
+                              <input type=Radio name="CC" value="Mastercard">Mastercard
+                              </p>
+                              <p>Credit card number
```
Comment (MazH-26) [inline indexBasecodeANSWER1c.html:62]: It could be worth adding the same standards you applied above by adding an id for the Credit card number element, so that it can be identified in testing. Also to close your paragraph in the same plave on the input type feild
